.github/labeler.yml

```diff
@@ -171,3 +171,7 @@
 'category: licensing':
 - 'licensing/**/*'
 - 'LICENSE'
+
+'category: conformance':
+- '**/layer_tests_summary/**'
+- '**/conformance/**'
```

Comment:
I suppose it's also covered by `IE Tests` label and conformance will not be filtered out

Reply:
This is to _skip_  some pipelines (like android) on changes only to these conformance-related folders